doc/source/cluster/cluster_under_construction/key-concepts.rst

```diff
@@ -1,130 +1,83 @@
+    on non-Kubernetes clusters).
 
 .. _cluster-autoscaler-under-construction:
 
 Autoscaler
 ----------
 
-The autoscaler is a process that runs on the :ref:`head node<cluster-head-node>`
-and is responsible for adding or removing :ref:`worker nodes<cluster-worker-node>`
-to meet the needs of the Ray workload while matching the specification in the
-:ref:`cluster config file<cluster-config>`. In particular, if the resource
-demands of the Ray workload exceed the current capacity of the cluster, the
-autoscaler will try to add nodes. Conversely, if a node is idle for long enough,
-the autoscaler will remove it from the cluster. To learn more about autoscaling,
-refer to the :ref:`Ray cluster deployment guide<deployment-guide-autoscaler>`.
-
-Ray Client
-----------
-The Ray Client is an API that connects a Python script to a remote Ray cluster.
-To learn more about the Ray Client, you can refer to the :ref:`documentation<ray-client>`.
-
-Job submission
---------------
+The autoscaler is a process that runs on the :ref:`head node<cluster-head-node-under-construction>`.
+It is responsible for provisioning or deprovisioning :ref:`worker nodes<cluster-worker-nodes-under-construction>`
+to meet the needs of the Ray workload. In particular, if the resource demands of the Ray workload exceed the
+current capacity of the cluster, the autoscaler will attempt to add more nodes. Conversely, if
+a node is idle for long enough, the autoscaler will remove it from the cluster.
 
-Ray Job submission is a mechanism to submit locally developed and tested applications
-to a remote Ray cluster. It simplifies the experience of packaging, deploying,
-and managing a Ray application. To learn more about Ray jobs, refer to the
-:ref:`documentation<ray-job-submission-api-ref>`.
+To learn more about the autoscaler and how to configure it, refer to the following user guides:
 
-Cloud clusters
---------------
+* :ref:`Configuring Autoscaling on VMs<deployment-guide-autoscaler-under-construction>`.
+* :ref:`Configuring Autoscaling on Kubernetes<kuberay-index>` (TODO cade@ update this link.)
 
-If you’re using AWS, GCP, Azure (community-maintained) or Aliyun (community-maintained), you can use the
-:ref:`Ray cluster launcher<cluster-launcher>` to launch cloud clusters, which
-greatly simplifies the cluster setup process.
+.. _cluster-clients-and-jobs-under-construction:
 
-Cluster managers
+Clients and Jobs
 ----------------
+Ray provides two methods for running workloads on a Ray Cluster: the Ray Client, and Ray Job Submission.
 
-You can simplify the process of managing Ray clusters using a number of popular
-cluster managers including :ref:`Kubernetes<kuberay-index>`,
-:ref:`YARN<ray-yarn-deploy>`, :ref:`Slurm<ray-slurm-deploy>` and :ref:`LSF<ray-LSF-deploy>`.
+* **The Ray Client** enables interactive development by connecting a local Python script or shell to the cluster.
```

Comment:
A few things:
1. We should explain that it's also fine to just SSH to the cluster and run `ray.init`. I think this should go first since I imagine a lot of people will want to do this when they're trying things out.
2. We should probably de-emphasize Ray Client in favor of Ray jobs.

Basically, we should think about what the user's flow is going from laptop to cluster and try to order the docs based on that. For example:
developing local script on laptop -> set up a cluster -> try out small version of script on cluster -> deploy full script on cluster

Reply:
can mention "ssh into head node or (kubectl exec into ray head pod on kubernetes)"